static/src/styles/app.css

```diff
@@ -66,3 +66,16 @@
         border: none;
     }
   }
+
+  dt.dl-inline {
+    display: inline;
+  }
+  dt.dl-inline:before {
+    content:"";
+    display:block;
+  }
+  dt.dl-inline + dd {
+    display: inline-block;
+    margin-left: 0em;
+    clear:right;
+  }
```

Comment:
Can we have a class for these? As applying them to all elements with that tag might have unintended consequences?